cmd/server/main.go

```diff
@@ -61,8 +65,9 @@ func main() {
 	if err != nil {
 		log.Panic(err)
 	}
+
 	proxy := httputil.NewSingleHostReverseProxy(upstreamURL)
-	r.PathPrefix("/").Handler(proxy)
+	r.PathPrefix("/").Handler(&RequestIntercepter{proxy})
```

Comment:
would it be better to target the path prefix and method?

```suggestion
	r.PathPrefix("/installer"). Methods("POST").Handler(&RequestIntercepter{proxy})
	r.PathPrefix("/").Handler(proxy)
```